src/Command/CheckAllSchemaTemplatesNameFieldsCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jobcloud\SchemaConsole\Command;
+
+use Jobcloud\SchemaConsole\Helper\SchemaFileHelper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CheckAllSchemaTemplatesNameFieldsCommand extends Command
+{
+    private const TYPES_FOR_VALIDATION = [
+        'record',
+        'enum',
+        'fixed'
+    ];
+
+    private const REGEX_MATCH_FIRST_LETTER = '/^[A-Za-z_]/';
```

Comment:
naming suggestion: REGEX_MATCH_FIRST_CHARACTER_IS_LETTER
I don't think we would want that the name of a schema starts or ends with an underscore and I also think we don't want upper case letters: `/^[A-Za-z_]/` => `/^[a-z].*[a-z]$/`